RentACar/RentACar/CRUD/CarCRUD.cs

```diff
@@ -35,7 +35,11 @@ public Car Create(Car car)
             {
                 cars = DeserializeJsonToList(_jsonFile);
 
-                car.Id = cars.Last().Id + 1;
+                if ((cars.Count > 0))
+                    car.Id = (cars.Last().Id + 1);
+                else
+                    car.Id = 0;
```

Comment:
It's odd you use `0` for the first car. What's the idea behind that? Also see my note about the ID generation on the general feedback comment.

Reply:
I have no idea! i didn't work with db before so i think the idea of "0" comes from the start index of arrays. I'm going to read about Grid and if i don't change it i will change start **Id** in 1 ! Thanks!

If i wanted to answer the general comments that you left me on the PR, where i can do that? 
About those comments i agree, i'm going to working on it! And again thank you very much for taking your time to feedback me.

Reply:
Np. To answer those maybe you can add a general comment like I did, and make use of markdown quotes (and answer below each quote). Sample:

> You can’t cross the sea merely by standing and staring at the water!

Rabindranath Tagore

Reply:
Oh! Nice! thanks! 😃